OnlyConsoleAppSolution1

tavisca-agupta opened the pull request:
Reviewer @tavisca-sjain

Commit:
OnlyConsoleApp

tavisca-sjain commented on ConsoleCalculator/Calculator.cs:
```diff
@@ -1,14 +1,127 @@
+        }
+
+        public static double Calculate(double A,double B,string opertor)
+        {
```
Use switch case here instead of multiple if and else, this will improve performance of the program.

tavisca-agupta replied:
Since Iam using using a Boolean variable which changes its value at runtime.So switch is not supporting and throwing me error as follows `Cannot implicitly convert type 'bool' to 'string' (CS0029) [ConsoleCalculator]`.So right now Iam not changing the whole logic of program, but I will always keep your advice from now onwards.
[I have taken reference from here](https://www.whizlabs.com/blog/ocajp-switch/)

tavisca-sjain commented on ConsoleCalculator/Calculator.cs:
```diff
@@ -1,14 +1,127 @@
+                A=0;B=0;temp="";temp2="";Flag=false;
+            }
+
+            else if((keyPressed!="+" && keyPressed!="-" && keyPressed!="x" && keyPressed!="X" && keyPressed!="/") && Flag==false)                      //extract out first operand
```
initialize an array of operators supported by your calculator and directly check if the key is matching with any operator charater using Array.Contains method and also use
for eg - OperatorArray.Contains(key, StringComparions.InvariantCultureIgnoreCase ) to avoid dealing with case sensitivity of each operator and key, during comparions. Make sure you use the same array in below area of code as well.

tavisca-agupta replied:
Updated in Solution v1.1

tavisca-sjain reviewed (commented):
Otherwise looks all good to me.

tavisca-agupta commented:
Check

tavisca-agupta commented:
Chevk

Commit:
TravisCI Integrated

Commit:
Updated TravisBuildConfig

Commit:
TravisBuildConfig v1.1

tavisca-agupta commented:
Solution v1.1
1)Fix Error msg while Dividing the result with non-zero number
2)Added Proper Test Cases
3)Added .travis.yml for TravisCI Integration.

Commit:
TravisBuildConfig v1.2

Commit:
TravisBuildConfig v1.3

tavisca-agupta commented:
@tavisca-agupta .
